flanker/addresslib/address.py

```diff
@@ -745,6 +739,12 @@ def __eq__(self, other):
     def __repr__(self):
         return ''.join(['[', self.to_unicode().encode('utf-8'), ']'])
 
+    def __str__(self):
+        return ', '.join(str(addr) for addr in self.container)
+
+    def __unicode__(self):
+        return u', '.join(addr.to_unicode() for addr in self.container)
```

Comment:
Can we use `unicode(addr)` here so that it parallels the implementation for `__str__`?